index.html

```
@@ -1,39 +1,611 @@
-    <p>Successfully deployed on AWS</p>
-    <p class="timestamp">Deployed: 08/09/2025</p>
+<div id="app">
+
+  <!-- ── Header ── -->
+  <header class="app-header">
+    <div class="logo">
+      <div class="logo-icon">🕐</div>
+      <h1>ChronoWise</h1>
+    </div>
+    <div class="header-actions">
+      <button class="icon-btn" title="Notifications" onclick="requestNotificationPermission()">🔔</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)